requirements/coverage.txt

```diff
@@ -36,7 +36,7 @@ mypy-extensions==0.4.3
     # via
     #   black
     #   typing-inspect
-numpy==1.22.0
+numpy==1.21.5
```

Comment:
See above, but FYI we'd need to pin this in `coverage.in` or it'd just be undone by our weekly update job.